### PR TITLE
[Eve-7] Fix compilation error using gcc8

### PR DESCRIPTION
**Pull request by alja**
This PR is related to #9013

### CODE REVIEW

**Comment by alja** [comment]
@bellenot @Axel-Naumann Please check.

**Comment by bellenot** [comment]
@phsft-bot build with flags -DCMAKE_CXX_STANDARD=17 -Droot7=ON

File: graf3d/eve7/src/REveManager.cxx [hunk 1]
```diff
@@ -1109,11 +1109,7 @@ void REveManager::GetServerStatus(REveServerStatus& st)
 {
    std::unique_lock<std::mutex> lock(fServerState.fMutex);
    gSystem->GetProcInfo(&fServerStatus.fProcInfo);
-#if defined(R__LINUX) || defined(_MSC_VER)
-   std::timespec_get(&fServerStatus.fTReport, TIME_UTC);
-#else
    fServerStatus.fTReport = std::time_t(nullptr);
```
Comment (bellenot) [inline graf3d/eve7/src/REveManager.cxx:1112]: That probably doesn't work on Windows (let's see the PR build)

Comment (alja) [reply graf3d/eve7/src/REveManager.cxx:1112]: @bellenot
Thx, I will check the result and add the _MSC_VER if there are errors.
Is Windows build triggered automatically?

**Comment by bellenot** [comment]
@alja here is the error:
```
REveManager.cxx(1115,48): error C2440: '<function-style-cast>': cannot convert from 'nullptr' to 'time_t' [C:\Users\bellenot\build\master\release\graf3d\eve7\ROOTEve.vcxproj]
```
see also https://godbolt.org/z/8crcWjYdh

File: graf3d/eve7/src/REveManager.cxx (hunk 1 above)
Comment (bellenot) [reply graf3d/eve7/src/REveManager.cxx:1112]: Sorry @alja , I compiled locally and pasted the error below..

Comment (bellenot) [reply graf3d/eve7/src/REveManager.cxx:1112]: You could keep something like:
```
#if defined(_MSC_VER)
   std::timespec fTReport = {0, 0};
#else
   std::time_t   fTReport = 0;
#endif
```
and
```
#if defined(_MSC_VER)
   std::timespec_get(&fServerStatus.fTReport, TIME_UTC);
#else
   fServerStatus.fTReport = std::time_t(nullptr);
#endif
```

Comment (alja) [reply graf3d/eve7/src/REveManager.cxx:1115]: @bellenot I have added the change now.

**Comment by bellenot** [comment]
@phsft-bot build with flags -DCMAKE_CXX_STANDARD=17 -Droot7=ON

**Review by bellenot — APPROVED** [review]
The failures look unrelated, so good enough for me

**Comment by bellenot** [comment]
@phsft-bot build

**Pull request merged**